src/checks/duplicated_key.rs

```diff
@@ -21,17 +23,22 @@ impl Check for DuplicatedKeyChecker {
         let key = line.get_key()?;
 
         if self.keys.contains(&key) {
-            return Some(Warning::new(
-                line.clone(),
-                self.template.replace("{}", &key),
-            ));
+            return Some(Warning::new(line.clone(), self.message(&key)));
         }
 
         self.keys.insert(key);
         None
     }
 }
 
+impl Message for DuplicatedKeyChecker {
+    fn message(&self, key: &str) -> String {
```

Comment:
We already have the PR with these changes https://github.com/mgrachev/dotenv-linter/pull/138
Sorry, but I can't merge these changes, because the PR came first.
It’s not very polite to another developer.
Could you revert these changes?